code-challenges.js

```diff
@@ -7,42 +7,179 @@
 
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
+// If the temperature entered by the user is below boiling point then a message that the temperature value "is below boiling point" will be delivered.
+// If the temperature entered by the user is above boiling point then a message that the temperature value "is above boiling point" will be delivered.
+// If the temperature entered by the user is equal to boiling point then a message that the temperature value "is equal to boiling point" will be delivered.
+// Boiling point is 212 F
+
+// Declare the variable to be temperature equal to 212.
+var temp = 212
+
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 35
-// var temp = 350
-// var temp = 212
+// Declare temp the variable is equal to 35
+// Declare boilingPt the variable is equal to 212
+// If temp 35 is less than boilingPt 212
+   // Print "35 is below boiling point"
+// Else if temp 35 is greater than boilingPt 212
+   // Print "35 is above boiling point"
+// Else temp 35 is equal to boilingPt 212
+  // Print "35 is at boiling point"
 
+var temp = 35
+var boilingPt = 212
+if (temp < boilingPt) {
+  console.log (`${temp} is below boiling point`)
+} else if (temp > boilingPt) {
+  console.log (`${temp} is above boiling point`)
+} else if (temp = boilingPt) {
+  console.log (`${temp} is at boiling point`)
+}
+
+
+// Declare temp the variable is equal to 350
+// Take boilingPt variable previousy declared
+// If temp 350 is less than boilingPt 212
+   // print "350 is below boiling point"
+// Else if temp 35 is greater than boilingPt 212
+   // Print "350 is above boiling point"
+// Else temp 350 is equal to boilingPt 212
+  // Print "350 is at boiling point"
+
+var temp = 350
+var boilingPt = 212
+if (temp < boilingPt) {
+  console.log (`${temp} is below boiling point`)
+} else if (temp > boilingPt) {
+  console.log (`${temp} is above boiling point`)
+} else if (temp = boilingPt) {
+  console.log (`${temp} is at boiling point`)
+}
+
+// Declare temp the variable is equal to 212
+// Take boilingPt variable previousy declared
+// If temp 212 is less than boilingPt 212
+   // print "212 is below boiling point"
+// Else if temp 212 is greater than boilingPt 212
+   // print "212 is above boiling point"
+// Else temp 212 is equal to boilingPt 212
+  // Print "212 is at boiling point"
+
+var temp = 212
+var boilingPt = 212
+if (temp < boilingPt) {
+  console.log (`${temp} is below boiling point`)
+} else if (temp > boilingPt) {
+  console.log (`${temp} is above boiling point`)
+} else if (temp = boilingPt) {
+  console.log (`${temp} is at boiling point`)
+}
```

Comment:
Great work here! very robust setting up code for each condition! 
Luckily next assessment we'll have functions so you won't have to copy and paste it each time!